package/MDAnalysis/coordinates/TRZ.py

```diff
@@ -358,13 +358,21 @@ def _seek(self, nframes):
 
         .. versionadded:: 0.9.0
         """
-        maxi_l = long(maxint)
-
-        framesize = long(self._dtype.itemsize)
-        seeksize = framesize * nframes
+        # On python 2, seek has issues with long int. This is solve in python 3
+        # where there is no longer a distinction between int and long int.
+        if six.PY2:
```

Comment:
And as for without a large file, I think technically we could mock up a fake file that returns byte offsets as if it were a large file or something... And then raised an Error if it was given a long int or something?  I'll just check this manually I think :D